src/go/go.mod

```diff
@@ -22,19 +22,3 @@ require (
 	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.1.0
 	google.golang.org/protobuf v1.27.1
 )
-
```

Comment:
Actually @reginawang3495 can you revert this change? @markjen / @hcgatewood  I'm not sure why this keeps happening, but some unrelated Golang development seems to affect this file.

Reply:
I think when I delete the changes here + `magma.iml`, I need to import a new magma module 
![image](https://user-images.githubusercontent.com/29292203/135414360-4cfface5-9986-4853-847f-215266c116c1.png)

When I create a new magma module, these 2 files get modified (sometimes, I believe `~/magma/go.mod` also changes)

Reply:
I think when I import `magma` as a new module, this part gets deleted + `go.mod` 
```
module magma

go 1.13
```
gets created

Reply:
Hunter said I can leave for now